fe/fe-core/src/main/java/com/starrocks/journal/bdbje/BDBEnvironment.java

```diff
@@ -394,7 +424,7 @@ public boolean close() {
                 try {
                     db.close();
                 } catch (DatabaseException exception) {
-                    LOG.error("Error closing db {}", db.getDb().getDatabaseName(), exception);
+                    LOG.error("Error closing db", exception);
```

Comment:
Why remove the db name when logging the exception?

Reply:
+1

Reply:
in some cases, getDatabaseName() will throws exception.

Reply:
The cases that bdb throws InsufficientLogException and RollbackException

Reply:
> 

How about only print out the id of the db? There should be some simpler and safer way to identify db. As an open source project that can be privatized deployed, I think our log on exception should be as detailed as possible because those logs are all we can get from the users.

Reply:
Good, but we can only get the name through getDatabaseName() func. I have write a safe method to get the name in CloseSafeDatabase.